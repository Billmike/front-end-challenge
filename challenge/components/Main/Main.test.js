import React from 'react';
import { Main } from './';
import renderer from 'react-test-renderer';

it('should mount properly', () => {
  const rendered = renderer.create(<Main />).toJSON();
  expect(rendered).toMatchSnapshot();
});
