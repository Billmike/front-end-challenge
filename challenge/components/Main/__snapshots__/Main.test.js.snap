// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should mount properly 1`] = `null`;
